policyengine_us/tests/policy/baseline/gov/territories/pr/tax/income/credits/ctc/pr_ctc_phase_out.yaml

```
@@ -0,0 +1,17 @@
+- name: Income over threshold, joint
+  period: 2024
+  input:
+    adjusted_gross_income: 150_000
+    pr_gross_income: 7_000
+    ctc_phase_out_threshold: 150_000
```

```suggestion
    ctc_phase_out_threshold: 150_000
    filing_status: JOINT
```
Can you do the same for other types of filing_status? Just to make sure the code is working.